### PR TITLE
Catch no subject error early

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

**Comment by arokem** [comment]
Any chance to add a test of this?

File: AFQ/tests/test_api.py [hunk 1]
```diff
@@ -180,6 +180,23 @@ def test_AFQ_custom_tract():
 
 
 @pytest.mark.nightly2
+def test_AFQ_no_derivs():
```
Comment (arokem) [inline AFQ/tests/test_api.py:183]: Should this test really be marked as nightly? It probably runs pretty quickly (?)

Comment (36000) [reply AFQ/tests/test_api.py:183]: test_AFQ_init is also marked nightly. Initializing an afq object takes about a minute I think, and if we have a lot of these tests, they can add up. To reduce time to make afq object, see: https://github.com/yeatmanlab/pyAFQ/issues/448

Comment (arokem) [reply AFQ/tests/test_api.py:183]: OK - that makes sense. Merging!

**Pull request merged**